content/en/docs/reference/glossary/workload.md

```diff
@@ -1,28 +1,29 @@
 ---
-title: Workload
-id: workload
-date: 2019-02-12
-full_link: /docs/concepts/workloads/
+title: Workloads
+id: workloads
+date: 2019-02-13
+full_link: https://kubernetes.io/docs/concepts/workloads/
 short_description: >
-  A set of applications for processing information to serve a purpose that is valuable to a single user or group of users.
+   A workload is a framing term to refer to a set of APIs/resources/objects.
 
-aka:
+aka: 
 tags:
+- fundamental
+- core-object
 - workload
 ---
-A workload consists of a system of services or applications that can run to fulfill a
-task or carry out a business process.
+  A workload is a framing term to refer to a set of APIs/resources/objects.
 
-<!--more-->
+<!--more--> 
 
-Alongside the computer code that runs to carry out the task, a workload also entails
-the infrastructure resources that actually run that code.
+Workload is a framing term to refer to a set of APIs/resources/objects. Kubernetes performs the 
+deployment and updates the workload with the current state of the application. Workloads let you define the 
+rules for application scheduling, scaling, and upgrade.
+Workloads includes the Daemon Sets, Deployments, Jobs, Pods, Replica Sets, Replication Controllers, Stateful Sets.
```

Comment:
s/includes/include
s/Daemon Sets/DaemonSet
s/Replica Sets/ReplicaSet
s/Replication Controllers/ReplicationController
s/Stateful Sets/StatefulSet

If you're going this far, suggest adding the rest of the list. Or remove the list altogether.

It's also really not true that workload objects define rules for application scheduling, etc ... yes, scheduling happens to workload objects, but this wording is misleading. (Same holds true for scaling and upgrade)

Reply:
done